### ISSUE TITLE
README.md file issues

### ISSUE THREAD

**Issue by pfurbacher** [Pyknic/MapStream#1]
1. "convinient" should read "convenient".
2. `.collect(joining("\n"))` should read `.collect(Collectors.joining("\n"))`.

**Comment by Pyknic** [comment]
Thanks! I fixed the typo. Regarding joining, it is a matter of which code style you prefer. I added a line at the bottom explaining the difference.

**Comment by pfurbacher** [comment]
Indeed, it may be a matter of style to just write `joining(...)`. But the reason I suggested the fully qualified form in the example is that if someone new to this code plops it into an IDE, `joining` will be highlighted as wrong, with little clue as to how to remedy it. Eclipse's quick fix suggests creating a new method called `joining`. I'm quite familiar with the streams API but don't use it every day, but for a moment I was stumped -- I couldn't remember what I had to import, or how I should qualify it.

So, especially in examples, you want to give the user the best shot at being successful in running the example. In this case, it should read `Collectors.joining(...)` since the code snippet does not include the static `import` statement.  The IDE would have a better chance of generating the necessary import statement.

The note you added is sufficient to help those interested in exploring your framework.
